include/circt/Dialect/RTL/Combinatorial.td

```diff
@@ -94,6 +94,7 @@ def SubOp : UTBinRTLOp<"sub">;
 def MulOp : UTVariadicRTLOp<"mul", [Commutative]>;
 def DivOp : UTBinRTLOp<"div">;
 def ModOp : UTBinRTLOp<"mod">;
+def ShlOp : UTBinRTLOp<"shl">;
```

Comment:
`UTBinRTLOp` implies that the LHS and RHS of the binary operator are the same width.  We might eventually want to relax this for something like a dynamic shift.  That said, we can continue to discuss this and refine in subsequent patches.